docs/book/src/architecture/agents.md

```diff
@@ -1,17 +1,3 @@
-# Index
-<!-- vim-markdown-toc GFM -->
-
-* [Agents](#agents)
-* [Configure Agents](#configure-agent)
-* [Application agent](#application-agent)
-    * [Binding a Service](#binding-a-service)
-    * [Claiming a Service](#claiming-a-service)
-        * [Claiming from Primaza cluster](#claiming-from-primaza-cluster)
-        * [Claiming from Worker cluster](#claiming-from-worker-cluster)
-* [Service agent](#service-agent)
-    * [Service Discovery](#service-discovery)
-
-<!-- vim-markdown-toc -->
 # Agents
```

Comment:
The image under this section needs to be updated to `architecture-agents-detailed.png`.

Reply:
great catch, fixed!